src/NzbDrone.Core/NetImport/TMDb/TMDbRequestGenerator.cs

```diff
@@ -42,77 +40,89 @@ public virtual NetImportPageableRequestChain GetMovies()
             var tmdbParams = "";
             switch (Settings.ListType)
             {
-                case (int)TMDbListType.List:
+                case (int) TMDbListType.List:
```

Comment:
While you are at it could you also change it, so the API Key is added outside this switch statement? So if we need to change that key, we don't have to edit all of these lines.

Reply:
Sure thing 👍